Formula/tomcat.rb

```diff
@@ -22,6 +22,10 @@ def install
     (bin/"catalina").write_env_script "#{libexec}/bin/catalina.sh", JAVA_HOME: Formula["openjdk"].opt_prefix
   end
 
+  def post_install
+    ln_s etc/"tomcat", libexec/"config" unless (libexec/"config").exist?
```

Comment:
```suggestion
    (etc/"config").mkpath
    ln_s etc/"tomcat", libexec/"config" unless (libexec/"config").exist?
```

Reply:
> Do we really want all versions of tomcat to use the same directory for their configs? Will that work if I have multiple versions installed and have different config files for each version?

As far as I know, the configs are compatible. I have migrated from previous versions of tomcat and it was using previous config.

Generally if you upgrade on linux, you are asked if you want to keep the previous one, but I am not sure this can behave similarly.

Reply:
Let's use `pkgetc` so that we have unique config directories for each version, and then add `caveats` that explain where we've set up config files to be installed.

Reply:
done as suggested, I hope 😬